src/firefly/js/templates/fireflyviewer/FireflyViewer.js

```diff
@@ -49,7 +51,7 @@ export class FireflyViewer extends PureComponent {
         const views = LO_VIEW.get(props.views) || LO_VIEW.none;
         this.state = this.getNextState();
         startTTFeatureWatchers();
-        if (views.has(LO_VIEW.images) ) launchTableTypeWatchers();
+        if (views.has(LO_VIEW.images) ) startImagesLayoutWatcher();
```

Comment:
no, we have never had it. It assumes that one you have images it will keep watching

Reply:
it does almost make since to remove coverage showing/hidden in the tbl-xy and the tbl-img bi-view mode. The problem is if it is not part of the display and you wanted it, you would have to go back to the tri-view and click something like `Coverage left` and then return to the bi-view mode. That does not quite make sense of I did coverage showing/hidden.